test/test_test_helper.rb

```diff
@@ -0,0 +1,17 @@
+require "test_helper"
+
+class TestTestHelper < ActionDispatch::IntegrationTest
```

Comment:
Is this test case needed?
Can we cover this test case by our multiple gemfiles?

Reply:
It certainly had a feeling of strangeness in the test with gemfile as a condition,
I added a test to think that it was better to have a guarantee that the handler of puma was loaded.
However, I also delete this file as I know that I feel a sense of incompatibility.

Reply:
It sounds "test for test" to me.
I think that "test for test" isn't reasonable on cost and maintainability view...

Reply:
> It sounds "test for test" to me.

Thanks. 
I'm convinced.